ansible/roles/erwin-idm/templates/deployment.yml.j2

```
@@ -0,0 +1,67 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: erwinidm-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: erwinidm
+spec:
+  replicas: {{ ERWINIDM_REPLICAS|default(1, true) }}
+  strategy:
+    type: Recreate
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: erwinidm
+  template:
+    metadata:
+      labels:
+        app: erwinidm
+      annotations:
+        configmapHash: ""
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: erwinidm
+        image: {{ ERWINIDM_IMAGE_NAME }}:{{ ERWINIDM_IMAGE_TAG }}
+        ports:
+        - containerPort: {{ ERWINIDM_PORT }}
+          name: erwinidm
+          protocol: TCP
+        envFrom:
+        - configMapRef:
+            name: erwinidm-configmap
+        - secretRef:
+            name: erwinidm-secret
+        resources:
+        # find reasonable limits
+          limits:
+            cpu: {{ ERWINIDM_CPU_LIMITS|default("1000m", true) }}
+            memory: {{ ERWINIDM_MEMORY_LIMITS|default("1Gi", true) }}
+          requests:
+            cpu: {{ ERWINIDM_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ ERWINIDM_MEMORY_REQUESTS|default("512Mi", true) }}
+        readinessProbe:
+          httpGet:
+            path: /health/ready
+            port: {{ ERWINIDM_PORT }}
```

Missing settings for the test intervall of the redinessProbe
```suggestion
        readinessProbe:
          httpGet:
            path: /health/ready
            port: {{ ERWINIDM_PORT }}
          timeoutSeconds: 4
          failureThreshold: 3
          periodSeconds: 5
```

---

done